python/ray/tune/ray_trial_executor.py

```diff
@@ -157,6 +157,28 @@ def pause_trial(self, trial):
             self._paused[trial_future[0]] = trial
         super(RayTrialExecutor, self).pause_trial(trial)
 
+    def reset_trial(self, trial, new_config, new_experiment_tag):
+        """Resets a trials configuration (if supported)
```

Comment:
Spacing below is off, but:

```
"""Tries to invoke `Trainable.reset_config()` to reset the trial.

       Args:
            trial (Trial): Trial to be reset.
            new_config (dict): New configuration for Trial
                trainable.
            new_experiment_tag (str): New experiment name
                for trial.
         Returns:
            True if `reset_config` is successful else False.

Reply:
Also, `new_config` and `new_experiment_tag` should be optional, right? @ericl

Reply:
I'm not sure what resetting without a new config and name means, it seems required to me.

Reply:
I was just thinking if you just want to reset the runtime without changing either config or name, you can do so without needing to hold on to the defaults.